proxies/live/apiproxy/targets/ers-target.xml

```diff
@@ -78,6 +56,64 @@
             </Step>
         </Response>
     </PreFlow>
+    <Flows>
+        <Flow name="user-restricted-flow">
+            <Condition>(accesstoken.auth_type == "user")</Condition>
+            <Request><!--AUTHORISED_APPLICATION business function is not supported in user restricted flow --><Step>
+                    <Name>RaiseFault.403Forbidden</Name>
+                    <Condition>(request.header.nhsd-ers-business-function == "AUTHORISED_APPLICATION")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.RemoveAndAddUserIdHeader</Name>
+                </Step> <Step>
+                    <Name>AssignMessage.Swap.NHSD-eRS-On-Behalf-Of-User-ID</Name>
+                    <Condition>(request.header.NHSD-eRS-On-Behalf-Of-User-ID ~~ ".+")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.Swap.nhsd-end-user-organisation-ods</Name>
+                    <Condition>(request.header.nhsd-end-user-organisation-ods ~~ ".+")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.Swap.nhsd-ers-business-function</Name>
+                    <Condition>(request.header.nhsd-ers-business-function ~~ ".+")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.Swap.nhsd-ers-comm-rule-org</Name>
+                    <Condition>(request.header.nhsd-ers-comm-rule-org ~~ ".+")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.Swap.nhsd-ers-file-name</Name>
+                    <Condition>(request.header.nhsd-ers-file-name ~~ ".+")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.Swap.nhsd-ers-referral-id</Name>
+                    <Condition>(request.header.nhsd-ers-referral-id ~~ ".+")</Condition>
+                </Step> <Step>
+                    <Name>AssignMessage.Remove.x-request-id-header</Name>
+                </Step> {% if ALLOW_ECHO_TARGET | default(false) == true %} <Step>
+                    <Name>AssignMessage.SetEchoTarget</Name>
+                    <Condition>(request.header.echo)</Condition>
+                </Step> {% endif %} <Step>
+                    <!--This should always be the last Step - as it is just before the message is sent - so the initial request stays intact for as long as possible.
+                The Swapping of the Request Headers converts X-Correlation-ID to NHSD-Correlation-ID before sending to backend. -->
+                    <Name>AssignMessage.Swap.CorrelationHeader</Name>
+                </Step></Request>
+            <Response/>
+        </Flow>
+        <!-- App restricted not yet supported -->
+        <Flow name="app-restricted-flow">
+            <Condition>(accesstoken.auth_type == "app")</Condition>
+            <Request>
+                <Step>
+                    <Name>RaiseFault.400BadRequest</Name>
```

Comment:
As mentioned previously I wonder if a 403 would make more sense than a 400 in this scenario?

Reply:
I don't mind changing this but probably worth getting Adam's view on this

Reply:
Yeah I agree 403.

Reply:
Changed